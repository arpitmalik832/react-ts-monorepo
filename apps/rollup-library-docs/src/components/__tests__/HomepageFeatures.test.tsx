import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import HomepageFeatures from '../HomepageFeatures';

describe('HomepageFeatures unit tests', () => {
  it('snapshot test', () => {
    const component = render(<HomepageFeatures />);

    expect(component).toMatchSnapshot();
  });
});
