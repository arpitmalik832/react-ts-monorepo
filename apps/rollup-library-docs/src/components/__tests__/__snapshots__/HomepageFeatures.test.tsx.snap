// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomepageFeatures unit tests snapshot test 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="features"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col col--4"
            >
              <div
                class="text--center"
              >
                <div />
              </div>
              <div
                class="text--center padding-horiz--md"
              >
                <h3>
                  Easy to Use
                </h3>
                <p>
                  Docusaurus was designed from the ground up to be easily installed and used to get your website up and running quickly.
                </p>
              </div>
            </div>
            <div
              class="col col--4"
            >
              <div
                class="text--center"
              >
                <div />
              </div>
              <div
                class="text--center padding-horiz--md"
              >
                <h3>
                  Focus on What Matters
                </h3>
                <p>
                  Docusaurus lets you focus on your docs, and we'll do the chores. Go ahead and move your docs into the 
                  <code>
                    docs
                  </code>
                   directory.
                </p>
              </div>
            </div>
            <div
              class="col col--4"
            >
              <div
                class="text--center"
              >
                <div />
              </div>
              <div
                class="text--center padding-horiz--md"
              >
                <h3>
                  Powered by React
                </h3>
                <p>
                  Extend or customize your website layout by reusing React. Docusaurus can be extended while reusing the same header and footer.
                </p>
              </div>
            </div>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="features"
    >
      <div
        class="container"
      >
        <div
          class="row"
        >
          <div
            class="col col--4"
          >
            <div
              class="text--center"
            >
              <div />
            </div>
            <div
              class="text--center padding-horiz--md"
            >
              <h3>
                Easy to Use
              </h3>
              <p>
                Docusaurus was designed from the ground up to be easily installed and used to get your website up and running quickly.
              </p>
            </div>
          </div>
          <div
            class="col col--4"
          >
            <div
              class="text--center"
            >
              <div />
            </div>
            <div
              class="text--center padding-horiz--md"
            >
              <h3>
                Focus on What Matters
              </h3>
              <p>
                Docusaurus lets you focus on your docs, and we'll do the chores. Go ahead and move your docs into the 
                <code>
                  docs
                </code>
                 directory.
              </p>
            </div>
          </div>
          <div
            class="col col--4"
          >
            <div
              class="text--center"
            >
              <div />
            </div>
            <div
              class="text--center padding-horiz--md"
            >
              <h3>
                Powered by React
              </h3>
              <p>
                Extend or customize your website layout by reusing React. Docusaurus can be extended while reusing the same header and footer.
              </p>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
